packages/11ty/_layouts/entry.liquid

```diff
@@ -47,7 +47,7 @@ Entry content, including entry image and tombstone data
   <div class="quire-entry__content">
     <header class="quire-entry__header">
       <div class="container">
-        <h1 class="quire-page__header__title" id="{{ title | url }}">
+        <h1 class="quire-page__header__title" id="{{ title | slugify }}">
```

Comment:
@anderspollack @fchasen we might use `page.filePathStem` or `page.fileSlug` here to to prevent duplicates though remove the file extensions, perhaps implementing an `idSlug` filter, somthing like

```javascript
  const { dir, name } path.parse(page.fileSlug)
  return `${dir}/${name}`
```

```suggestion
        <h1 class="quire-page__header__title" id="{{ page.fileSlug | idSlug }}">
```

Reply:
This make sense to me, but we might want to append some text like "header_" to the file path in case it starts with a number.